styles.css

```diff
@@ -191,18 +188,26 @@ main {
   top: 9.5%;
   left: 23%;
   transform: rotate(9deg);
-  visibility: visible;
 }
 
 /* Clothes */
 .dress-clothes {
   position: absolute;
   z-index: 6;
+  width: 79%;
```

Comment:
Nice job getting all these garments in place! It sounds like it was difficult to get the positioning down as the screen size scaled down. Nice work!